cmd/simRecon/simRecon.go

```diff
@@ -52,32 +52,52 @@ func ReconstructSeq(newickInput string, fastaInput string, outputFilename string
 }
 
 //SimRecon simulates evolution, performs reconstruction, and then evaluates the accuracy of the reconstruction
-func SimRecon(rootFastaFile string, treeFile string, gp string, simOutFile string, leafOutFile string, reconOutFile string, accuracyOutFile string) {
+func SimRecon(rootFastaFile string, treeFile string, gp string, simOutFile string, leafOutFile string, reconOutFile string, accuracyOutFile string, option int, baseAccFile string) {
 	//TODO: this code will need to change drastically for sequences of varying lengths.
 	//The loop through the sequence is restricted by the length of a single fasta and the tot calculation will need to calculate the total number of bps
 	//ReconAccuracy calculates the total number of incorrectly reconstructed base pairs in a tree and returns a percentage of correct base calls
 	SimulateEvolve(rootFastaFile, treeFile, gp, simOutFile, leafOutFile)
 	ReconstructSeq(treeFile, leafOutFile, reconOutFile)
 
-	answer := reconstruct.ReconAccuracy(simOutFile, reconOutFile, leafOutFile, gp)
+	answer, byBaseAnswer := reconstruct.ReconAccuracy(simOutFile, reconOutFile, leafOutFile, gp, option)
 	out := fileio.EasyCreate(accuracyOutFile)
 	defer out.Close()
 
 	for name, accuracy := range answer {
 		fmt.Fprintf(out, "%s\t%f\n", name, accuracy)
 	}
+	if option == 1 && baseAccFile != "" {
```

Comment:
Maybe option can be removed as a variable and you only need to test if baseAccFile != ""?